### PR TITLE
Add --colorize and --nocolorize options to IRB

### PR DESCRIPTION
**Pull request by aycabta**
@k0kubun Please review this.

### CODE REVIEW

**Comment by k0kubun** [comment]
By this change `use_colorize?` becomes nil by default, right? Why not set `@CONF[:USE_COLORIZE] = true` in `init_config`?

**Comment by k0kubun** [comment]
Also please check CI status too:

```
  1) Error:
TestIRB::TestWorkSpace#test_code_around_binding:
NoMethodError: undefined method `conf' for IRB:Module
    /home/travis/build/ruby/ruby/lib/irb/workspace.rb:134:in `code_around_binding'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:22:in `block (2 levels) in test_code_around_binding'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:98:in `without_term'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:22:in `block in test_code_around_binding'
    /home/travis/build/ruby/ruby/lib/tempfile.rb:336:in `create'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:10:in `test_code_around_binding'
  2) Error:
TestIRB::TestWorkSpace#test_code_around_binding_with_script_lines__:
NoMethodError: undefined method `conf' for IRB:Module
    /home/travis/build/ruby/ruby/lib/irb/workspace.rb:134:in `code_around_binding'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:59:in `block (3 levels) in test_code_around_binding_with_script_lines__'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:98:in `without_term'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:59:in `block (2 levels) in test_code_around_binding_with_script_lines__'
    /home/travis/build/ruby/ruby/lib/tempfile.rb:336:in `create'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:54:in `block in test_code_around_binding_with_script_lines__'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:87:in `with_script_lines'
    /home/travis/build/ruby/ruby/test/irb/test_workspace.rb:53:in `test_code_around_binding_with_script_lines__'
```

**Comment by aycabta** [comment]
@k0kubun Please review again.

File: lib/irb.rb [hunk 1]
```diff
@@ -96,6 +98,7 @@
 #     IRB.conf[:BACK_TRACE_LIMIT]=16
 #     IRB.conf[:USE_LOADER] = false
 #     IRB.conf[:USE_READLINE] = nil
+#     IRB.conf[:USE_COLORIZE] = nil
```
Comment (k0kubun) [inline lib/irb.rb]:
```suggestion
#     IRB.conf[:USE_COLORIZE] = true
```
might be more friendly to indicate it expects boolean values.

**Comment by aycabta** [comment]
[Merged.](https://github.com/ruby/ruby/commit/3a9008b9845db544d0788851f9527ce9f9f6bf4a)

**Pull request closed without merging**